### PR TITLE
extend release instructions

### PR DESCRIPTION
**Pull request by steveeJ**
### TODO:
- [x] CHANGELOG(s) updated with appropriate info
- [x] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

File: crates/release-automation/README.md [hunk 1]
```diff
@@ -52,15 +58,37 @@ Automated steps still require running the tool manually ;-).
...
+    export HOLOCHAIN_REPO=$(mktemp -d)
+    export HOLOCHAIN_NIXPKGS_REPO=$(mktemp -d)
+    export HOLONIX_REPO=$(mktemp -d)
+    ```
+
+0. _[M]_ Prepare the holochain repository and enter it.
+
+    ```sh
+    git clone "${HOLOCHAIN_URL}" "${HOLOCHAIN_REPO}"
+    pushd "${HOLOCHAIN_NIXPKGS_REPO}"
```
Comment (jost-s) [inline crates/release-automation/README.md:77]: this must be `pushd "${HOLOCHAIN_REPO}"`

Comment (steveeJ) [reply crates/release-automation/README.md]: thanks!

File: crates/release-automation/README.md [hunk 2]
```diff
@@ -69,70 +110,243 @@ Automated steps still require running the tool manually ;-).
...
+    0. Drop the commit made by the _BumpReleaseVersions_ step using `git rebase -i HEAD~1`.
+    0. Make the required changes and commit them.
+    0. Repeat the _BumpReleaseVersions_ step.
+    0. Continue the process from there...
+
+0. _[M]_ Add additional release-specific environment variables:
 
     ```sh
-    git push -u upstream $(git branch --show-current)
+    export TAG=$(git tag --list | tac | grep -m1 holochain-)
```
Comment (jost-s) [inline crates/release-automation/README.md:150]: `tac` doesn't exist on macOS by default. Can we replace this by
`export TAG=$(git tag --list | tail -r | grep -m1 holochain-)`

File: crates/release-automation/README.md [hunk 3]
```diff
@@ -69,70 +110,243 @@ Automated steps still require running the tool manually ;-).
...
 
-0. _[M]_ Create a Pull-Request from the release branch to the main branch
+0. _[M]_ Push the release branch. Example:
+
+    ```sh
+    git push -u origin $(git branch --show-current)
+    ```
 
-0. _[M]_ Ensure the CI tests pass
+0. _[M]_ Create a Pull-Request from the release branch to the main branch and ensure the CI tests pass
```
Comment (jost-s) [inline crates/release-automation/README.md:162]: Although it doesn't say `main` branch but main branch, it's clearer to refer to the `develop` branch.

Comment (jost-s) [reply crates/release-automation/README.md:162]: Nevermind.

File: crates/release-automation/README.md [hunk 4]
```diff
@@ -69,70 +110,249 @@ Automated steps still require running the tool manually ;-).
...
+        [${VERSION_COMPAT}]
+        git-src = "revision:${TAG}"
+        lair-version-req = "~0.0"
+        EOF
+
+        # add the new tag to the CI config so it's built and cached by it
+        nix-shell -p yq-go --run 'yq e -i \
+          ".jobs.holochain-binaries.strategy.matrix.nixAttribute = .jobs.holochain-binaries.strategy.matrix.nixAttribute + [\"${VERSION_COMPAT}\"]" \
+          .github/workflows/build.yml'
+                nix run nixpkgs.yq-go -c yq e -i \
```
Comment (jost-s) [inline crates/release-automation/README.md:211]: this is the old command

File: crates/release-automation/README.md [hunk 5]
```diff
@@ -69,70 +110,238 @@ Automated steps still require running the tool manually ;-).
...
 
-0. _[M]_ Push the release branch. Example:
+    ***NOTE***: If at any point after this any changes need to be made to the code for this release, please come back here and follow these steps:
+
+    0. Drop the commit made by the _BumpReleaseVersions_ step using `git rebase -i HEAD~1`.
+    0. Make the required changes and commit them.
+    0. Repeat the _BumpReleaseVersions_ step.
+    0. Continue the process from there...
+
+0. _[M]_ Add additional release-specific environment variables:
```
Comment (jost-s) [inline crates/release-automation/README.md:151]: @steveeJ I suggest to append these env vars to `~/.holochain_release.sh`.

**Review by jost-s — APPROVED** [review]
No further comments other than the above.

**Comment by steveeJ** [comment]
> No further comments other than the above.

thank you for working through this :raised_hands: 

PR is updated and rebased. please take a final look :pray:

**Review by jost-s — APPROVED** [review]
optimal

**Comment by steveeJ** [comment]
despite putting `skip-checks: true` in the commit message githab ran the CI checks. it doesn't make sense to block this PR on the failures.

**Pull request merged**